MrFish.lua

```diff
@@ -149,39 +149,40 @@ function addon:GetFishingPole()
 		end
 	end
 	if (maxname) then
-		self:Print(format(L["Found %s"],maxname))
+		self:Print(format(L["Best Fishing Pole: %s"],maxname))
```

Comment:
In order to change a dictinaly value, plese customize localization instead of changing the keyword

Reply:
There is no localisation in my localization.lua.
Maybe it will be handled at Curseforge.
I'll restore the Original Value.